examples/attention/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# The atom_types.h file is written to the build directory
+
+# PLNDynamicExpSetupModule
+INCLUDE_DIRECTORIES(${CMAKE_BINARY_DIR})
+
+IF (TBB_FOUND)
```

Comment:
I ask this, because in my experience, using these non-standard vendor libraries tend to induce snake-bites down the line.  Look at the fiasco that is boost for a shining examples of bad code.